script.js

```diff
@@ -22,3 +24,248 @@ for (let i = 0; i < menuLinks.length; i += 1) {
+  if (e.target.classList.contains('project-btn')) {
+    const modalBackground = document.createElement('div');
+    const modalBoxContent = `
+    <div class="modal-box">
+    <div class="modal-inner">
+      <div class="modal-title" >
+        <div class="modal-header">
+          <h3>Keeping track of hundreds of components website</h3>
+          <div>
+            <img src="./assets/Icon.png" class="close-menu" alt="close-menu" />
+          </div>
+        </div>
+        <br>
+        <ul class="modal-list">
+          <li class="tools-1">HTML</li>
+          <li class="tools-1">Boostrap</li>
+          <li class="tools-1">Ruby on rails</li>
+        </ul>
+      </div>
+      <div class="modal-img-box">
+        <div class="modal-img">
+          <img src="/assets/Snapshoot Portfolio.png" alt="" />
+        </div>
+
+        <div class="modal-details">
+         <div class="modal-info">
+          <p>
+            Lorem Ipsum is simply dummy text of the printing and typesetting
+            industry. Lorem Ipsum has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a galley of
+            type and scrambled it 1960s. 
+           </p>
+
+           <p>
+            Lorem Ipsum is simply dummy text of
+            the printing and typesetting industry. Lorem Ipsum has been the
+            industry's standard dummy text ever since the 1500s, when an
+            unknown printer took a galley of type and scrambled it 1960s with
+            the releax map lapora verita.
+           </p>
+         </div>
+         
+         <div class="modal-btns">
+          <div class="see-live">
+            <a href="#">See Live  <img src="/assets/Icon-see live.svg" alt=""></a>
+          </div>
+          <div class="see-source">
+            <a href="#">See Source  <img src="/assets/Vector.png" alt=""></a>
+          </div>
+         </div>
+        </div>
+      </div>
+    </div>
+  </div>
+    `;
+    modalBackground.setAttribute('class', 'modal-background');
+    modalBackground.innerHTML = modalBoxContent;
+    workSection.appendChild(modalBackground);
+    document.body.style.overflowY = 'hidden';
+  }
+
+  if (e.target.classList.contains('btn-bg')) {
+    const modalBackground = document.createElement('div');
+    const modalBoxContent = `
+   <div class="modal-box">
+   <div class="modal-inner">
+     <div class="modal-title" >
+       <div class="modal-header">
+         <h3>Keeping track of hundreds of components website</h3>
+         <div>
+           <img src="./assets/Icon.png" class="close-menu" alt="close-menu" />
+         </div>
+       </div>
+       <br>
+       <ul class="modal-list">
+         <li class="tools-1">HTML</li>
+         <li class="tools-1">Boostrap</li>
+         <li class="tools-1">Ruby on rails</li>
+       </ul>
+     </div>
+     <div class="modal-img-box">
+       <div class="modal-img">
+         <img src="/assets/Snapshoot Portfolio.png" alt="" />
```

Comment:
:red_circle: As you can see from the screenshots below Your design lacks the image on modal open. Please sure the appropriate  image of the selected card displays to match the Figma design

Your design

![Screenshot from 2023-05-27 13-09-46](https://github.com/Adesina-Abiodun/Microverse-Portfolio-Project/assets/26736582/3a7a592e-25ea-439c-84d0-9d7a02991b92)

Figma
![Screenshot from 2023-05-27 13-09-34](https://github.com/Adesina-Abiodun/Microverse-Portfolio-Project/assets/26736582/315a739e-4afb-4b13-a549-12e7ca2c9f01)